src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -227,24 +227,23 @@ public function storeOutput()
         ) {
             $outputFilesConfig = $this->storageConfig["output"]["files"];
         }
-
+        if (isset($this->storageConfig["output"]["table_files"]) &&
+            count($this->storageConfig["output"]["table_files"])
+        ) {
+            $outputTableFilesConfig = $this->storageConfig["output"]["table_files"];
+        }
         $this->logger->debug("Uploading output tables and files.");
 
         $uploadTablesOptions = ["mapping" => $outputTablesConfig];
 
         $systemMetadata = [
             TableWriter::SYSTEM_KEY_COMPONENT_ID => $this->component->getId(),
             TableWriter::SYSTEM_KEY_CONFIGURATION_ID => $this->configId,
+            TableWriter::SYSTEM_KEY_RUN_ID => $this->clientWrapper->getBasicClient()->getRunId(),
```

Comment:
this was previously only being included on feature presence

Reply:
similarly, this will be added to tables too, which i'm not sure is of any value, it's bound to change on every rune and storage jobs are better way to pair those together

Reply:
right, maybe we need to differentiate based on the destination for these.  I'll try an edit that I hope makes sense.

Reply:
you can add it in the if ($this->useFileStorageOnly()) { code branch?

Reply:
I switched it so that tables will always include branchId if present, and files will include runId